### PR TITLE
Stale too many redirects (batch 1)

### PR DESCRIPTION
**Pull request by jayvdb**
Re-enables 19 rulesets which were disabled due to too-many-redirects which is no longer accurate.

Related to https://github.com/EFForg/https-everywhere/issues/18981

The failing rules in the "fetch" job all pass locally when running the checker manually.

### CODE REVIEW

**Comment by jayvdb** [comment]
I removed the rulesets which were failing in https://travis-ci.org/EFForg/https-everywhere/jobs/660136038 ; again they all worked for me locally, but I'll defer them to the next batch.

I can split the commit "Lots of too-many-redirects fixes" into nine separate commits if requested, one for each ruleset.

If it helps I can also move to a new PR the few changes in this PR which are not re-enabling a ruleset, as that is a reasonable division in this problem.

File: src/chrome/content/rules/Sensiolabs.com.xml [hunk 1]
```diff
@@ -1,25 +1,14 @@
...
     <target host="www.sensiolabs.com" />
-    <target host="support.sensiolabs.com" />
-    <target host="www.support.sensiolabs.com" />
+    <target host="blog.sensiolabs.com" />
     <target host="insight.sensiolabs.com" />
     <target host="www.insight.sensiolabs.com" />
     <target host="connect.sensiolabs.com" />
-    <target host="www.connect.sensiolabs.com" />
 
     <securecookie host="^(www\.)?(support\.|insight\.|connect\.)?sensiolabs\.com" name=".+" />
```
Comment (J0WI) [inline src/chrome/content/rules/Sensiolabs.com.xml]:
```suggestion
    <securecookie host=".+" name=".+" />
```

File: src/chrome/content/rules/DMLP.org.xml [hunk 2]
```diff
@@ -1,36 +1,23 @@
...
+	<test url="http://dmlp.org/favicon.ico" />
+	<test url="http://www.dmlp.org/favicon.ico" />
 
-	<rule from="^http://(?:www\.)?citmedialaw\.org/"
-		to="https://www.citmedialaw.org/" />
-
-	<rule from="^http://(?:www\.)?dmlp\.org/(?!favicon\.ico|files/|misc/|modules/|sites/)"
-		to="https://www.dmlp.org/" />
+	<rule from="^http://(?:www\.)?([^/]+)/"
+		to="https://www.$1/" />
```
Comment (J0WI) [inline src/chrome/content/rules/DMLP.org.xml]: You can just use multiple simple redirects here.

Comment (jayvdb) [reply src/chrome/content/rules/DMLP.org.xml]: I used this per https://github.com/EFForg/https-everywhere/issues/18910

If you still want me to change, should I be using four literal or two basic rules here?

Comment (jayvdb) [reply src/chrome/content/rules/DMLP.org.xml:21]: ping @J0WI

Comment (J0WI) [reply src/chrome/content/rules/DMLP.org.xml:21]: I prefer two basic rules like in the example [here](https://github.com/EFForg/https-everywhere/issues/18910#issuecomment-600088525)

Comment (J0WI) [reply src/chrome/content/rules/DMLP.org.xml:21]: ping @jayvdb

**Pull request closed without merging**